### ISSUE TITLE
[tmux] sometimes window fails to open

### ISSUE THREAD

**Issue by 130s** [130s/compenv_ubuntu#3]
With the setting like this:

```
pane1 | pane2
------------------
      pane3
```
Running the following from pane2:
```
$ emacs &
[1] 29722
** (emacs:29722): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-6hbbbGBgHw: Connection refused
```

**Comment by 130s** [comment]
Happened with `meld` too.

```
$ meld . &                                                                                                                  
[1] 4657
$ /usr/lib/python2.7/dist-packages/gtk-2.0/gtk/__init__.py:57: GtkWarning: could not open display
  warnings.warn(str(e), _gtk.Warning)
/usr/bin/meld:155: GtkWarning: IA__gtk_icon_theme_get_for_screen: assertion 'GDK_IS_SCREEN (screen)' failed
  gtk.icon_theme_get_default().append_search_path(meld.paths.icon_dir())
Traceback (most recent call last):
  File "/usr/bin/meld", line 155, in <module>
    gtk.icon_theme_get_default().append_search_path(meld.paths.icon_dir())
AttributeError: 'NoneType' object has no attribute 'append_search_path'
```

I found that this can be worked around by the following for each terminal:
```
$ export DISPLAY=:0 
```

Not sure but I ran [these commands](http://stackoverflow.com/a/3445216/577001) prior to the fix above:
```
sudo apt-get remove --purge xserver-xorg
sudo apt-get install xserver-xorg
sudo dpkg-reconfigure xserver-xorg
```

Not yet sure if I should add `export DISPLAY=:0` to bash config. So for now I keep this ticket open.

**Comment by 130s** [comment]
On freshly installed machine,

```
$ meld . &
[1] 11128
n130s@tork-kudu1:~/link/github_repos/130s/compenv_ubuntu/config/bash$ 
(meld:11128): GConf-WARNING **: Client failed to connect to the D-BUS daemon:
Failed to connect to socket /tmp/dbus-zduub8N9uZ: Connection refused

(meld:11128): GConf-WARNING **: Client failed to connect to the D-BUS daemon:
Failed to connect to socket /tmp/dbus-zduub8N9uZ: Connection refused
Traceback (most recent call last):
  File "/usr/bin/meld", line 178, in <module>
    main()
  File "/usr/bin/meld", line 164, in main
    already_running, dbus_app = meld.dbus_service.setup(app)
  File "/usr/lib/meld/meld/dbus_service.py", line 56, in setup
    bus = dbus.SessionBus()
  File "/usr/lib/python2.7/dist-packages/dbus/_dbus.py", line 211, in __new__
    mainloop=mainloop)
  File "/usr/lib/python2.7/dist-packages/dbus/_dbus.py", line 100, in __new__
    bus = BusConnection.__new__(subclass, bus_type, mainloop=mainloop)
  File "/usr/lib/python2.7/dist-packages/dbus/bus.py", line 122, in __new__
    bus = cls._new_for_bus(address_or_type, mainloop=mainloop)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NoServer: Failed to connect to socket /tmp/dbus-zduub8N9uZ: Connection refused
```

**Comment by 130s** [comment]
Should be fixed by #48 
If not be sure to reopen.
